In [1]:
import pandas as pd
import numpy as np
from scipy import stats
from scipy.stats import norm
from scipy.stats import chi2_contingency
from scipy.stats import chi2
import seaborn as sns

In [2]:
df = pd.read_csv("C:/Users/VISHAL S PANCHANGE/Downloads/Cutlets.csv")
df.head()

Unit A  Unit B
0  6.8090  6.7703
1  6.4376  7.5093
2  6.9157  6.7300
3  7.3012  6.7878
4  7.4488  7.1522

### Null hypothesis: There is no significant difference in the diameter of the cutlet between the two units.
### Alternative hypothesis: There is a significant difference in the diameter of the cutlet between the two units.

In [5]:
t_statistic, p_value = stats.ttest_ind(df['Unit A'], df['Unit B'])

# print results
print('t-statistic:', t_statistic)
print('p-value:', p_value)


t-statistic: 0.7228688704678063
p-value: 0.4722394724599501


### Pvalue (0.4722394724599501) is greater tha  alpha(0.05) we accept Alternative hypothesis